### ISSUE TITLE
Add missing getter for unsigned long long and long long, in "ToXXX" family of methods

### ISSUE THREAD

**Issue by petersvp** [Zguy/Jzon#25]
Add missing getter for unsigned long long and long long, in "ToXXX" family of methods.
Currently there are none, i had to manually add them

**Comment by ghost** [comment]
I don't think this project is being actively maintained by the owner anymore. Last commit was late 2015.
